docs/supported-languages.md

```diff
@@ -76,10 +137,123 @@ The following thresholds define each maturity level:
 
 Visit the cheat sheet generation script and associated semgrep-core test files to learn more about each feature:
 
-* [generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
-* [semgrep-core test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
-## Language parse rates
+* [Generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
+* [`semgrep-core` test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
+
+### Language parse rates
+
+See [Parse rates by language](https://dashboard.semgrep.dev/).
+
+## Semgrep Supply Chain
+
+<SscIntro/>
+
+Semgrep Supply Chain parses **lockfiles** for dependencies, then scans your codebase for reachable findings based on the lockfiles. Some languages, such as Java, have several lockfiles, depending on your repository's package manager. For some languages, such as Javascript and Python, a manifest file is also parsed.
+
+The following tables list lockfiles, their package manager and language, and their support levels.
+
+### General Availability
+
+<table>
+<thead><tr>
+    <th>Language</th>
+    <th>Supported package managers</th>
+    <th>Lockfile</th>
+</tr></thead>
+<tbody><tr>
+   <td>Go</td>
+   <td>Go modules (<code>go mod</code>)</td>
+   <td><code>go.sum</code></td>
+  </tr>
+  <tr>
+   <td rowspan="2">JavaScript / TypeScript</td>
+   <td>npm (Node.js)</td>
+   <td><code>package-lock.json</code></td>
+  </tr>
+  <tr>
+   <td>Yarn, Yarn 2, Yarn 3</td>
```

Comment:
Should we say 'Yarn (all versions)' instead so it's easier on the eyes?

Reply:
i wrote out all the names so that people can search for them :)

Reply:
@andy-r2c Reply with any emoji if this works for you 😺